src/psyclone/psyGen.py

```diff
@@ -2489,7 +2689,8 @@ def update(self):
         if not parent:
             raise InternalError("Failed to find parent node in which to "
                                 "insert OpenMP parallel do directive")
-        start_idx = parent.content.index(self._children[0]._ast)
+        start_idx = object_index(parent.content,
+                                 self._children[0]._ast)
```

Comment:
pylint:

W:2693,33: Access to a protected member _ast of a client class (protected-access)

Reply:
Fixed.